### ISSUE TITLE
Dictionary updates

### ISSUE THREAD

**Issue by katewhalen** [freebee-game/enable#40]
Please add the following words:

Lede
Ledes
Meme
Memes
Racecar
Racecars
Stent
Stents
Stenting
Ramen
Spam

**Comment by retroodux** [comment]
Ledes is not in common usage, plural form no. There is only one lede/the lede re: an article 

Stenting I don't love. Verbing a noun that has already taken on verb form "stent" but you don't conjugate a verbed noun necessarily 

 Spam  brand names. 

Ramen I agree is no longer only foreign lang yeah

My preferences anyway thanks for letting me hop on comments

**Comment by katewhalen** [comment]
Hi,

Thanks for your comments.   Here are a few responses.

Ledes is a perfectly valid word, whether in common usage or not.  If we
want to use frequency of usage as the standard for inclusion in this
dictionary (and puzzle solution bank), well, I've got a list of words as
long as my arm that I'd like to request you remove! :D  Google the word
"lede":

noun: *lede*; plural noun: *ledes*

   1. the opening sentence or paragraph of a news article, summarizing the
   most important aspects of the story.

If there are multiple articles, then you have multiple ledes.

Re: stenting, one could argue how much specific medical terminology to
include in such a dictionary, but I will leave you with a quote from
Wikipedia:

"In medicine <https://en.wikipedia.org/wiki/Medicine>, a *stent* is a metal
or plastic tube inserted into the lumen
<https://en.wikipedia.org/wiki/Lumen_(anatomy)> of an anatomic vessel or
duct to keep the passageway open, and *stenting* is the placement of a
stent." (https://en.wikipedia.org/wiki/Stent)

Re: spam, the lowercase spelling is very common usage; even if it does
originate from a brand name, it is clearly not being used in that context.
"noun
(lowercase)Digital Technology. disruptive online messages, especially
commercial messages posted on a computer network or sent as email (often
used attributively): Install spam blocker software and keep your email spam
filters updated to protect your accounts from unsolicited spam." (
https://www.dictionary.com/browse/spam).

Re: ramen, I would hope that this would be included.  I have noticed quite
a few words of Spanish origin popping up in the solutions and so I am
assuming that the main contributors to the dictionary are American.  It
seems only fair to include loan words from other languages, too.
